tests/package.json

```diff
@@ -6,17 +6,16 @@
   "author": "Owais Lone",
   "license": "MIT",
   "devDependencies": {
-    "babel": "^5.4.7",
-    "babel-core": "^5.4.7",
-    "babel-loader": "^5.1.3",
-    "css-loader": "^0.14.1",
-    "extract-text-webpack-plugin": "^0.8.0",
-    "node-libs-browser": "^0.5.0",
-    "react": "^0.13.3",
-    "style-loader": "^0.12.3",
-    "webpack": "^1.9.8",
-    "webpack-bundle-tracker": "0.0.8",
-    "webpack-dev-server": "^1.9.0",
-    "webpack-split-by-path": "0.0.1"
+    "@babel/core": "^7.9.6",
```

Comment:
The PR is there for a while, some packages were updated in the npm registry. Should we upgrade them?

Reply:
The purpose of this PR is to test the new version of `webpack-bundle-tracker` which is in alpha state right now. To me it makes sense trying this integration and set the `django-webpack-loader` in alpha state. Once this is know to be working, bump the dependencies during the official new release. Is that ok?

Reply:
You are right, it is not a concern for this PR :)

Sorry about that.